### ISSUE TITLE
Data (Feature)

### ISSUE THREAD

**Issue by kriskuiper** [cmda-bt/be-course-18-19#43]
# Data

## Name
Kris Kuiper

## Repo link
[My Project Tech repo](https://github.com/kriskuiper/project-tech)

## Summary
I made it possible for users to add their own posts via a form with a title, description etc. Also there's a detail page for every post.

## Resources used
* [The `express-server` example repo](https://github.com/cmda-bt/be-course-18-19/blob/master/examples/express-server/index.js)

## Any thoughts?
I found it surprisingly easy to implement the data out of the form right in to the posts. I even added some more CSS finally to make it look a bit more appealing to the eye. I still have to implement the logic so the user is possible to upload images to their post and make that images show up at the detail page.

**Comment by rijkvanzanten** [comment]
Great job!

I tried looking for things to hate on, but the worst I've been able to find were some out of place empty lines 😁 https://github.com/kriskuiper/project-tech/blob/master/app/models/fakedata.js#L12, https://github.com/kriskuiper/project-tech/blob/master/app/index.js#L16, https://github.com/kriskuiper/project-tech/blob/master/app/index.js#L34

Great work on code consistency, keeping everything lint free, and repo organization.

Oh yeah a small note: `docs` is usually used to refer to "documentation", not "documents", in open source repos. I expected to find more info on how to use or work on your app in the `/docs` folder, but instead you store the contributing and license files there. I would recommend keeping those on the same level as the `README.md` file. (It also allows GitHub to read the LICENSE file, and display it in the top right :) )
